test/compilable/imports/cstuff1.c

```diff
@@ -330,6 +330,17 @@ int tests4()
     return p->b;
 }
 
+int test5()
+{
+    struct { };
```

Comment:
This line does not compile (`error abstract declarator...`) with e.g. GCC 10.2 on godbolt, the rest of the function is fine. You should probably add a test to make sure it doesn't

ImportC manages to compile a bunch of C that GCC correctly errors on.  @ibuclaw can probably give you a list.

Reply:
The C11 standard says that is acceptable code. I read it fairly carefully, but if you can show me it says otherwise, I'll change it. The policy on ImportC is to accept any code that C11 says is acceptable, with no warnings.

Reply:
@thewilsonator I already know that cstuff1.c is not compilable by neither gcc nor clang.  The problem you need to isolate though is whether it is rejected by the parser, or by semantic?

Reply:
Ah you are correct, I had opened up a C++ panel without realising it, Carry on.

Reply:
To wit: C11 6.7.2.1-8 sez:

"If the struct-declaration-list does not contain any named members, either directly or via an anonymous structure or anonymous union, the behavior is undefined."

More particularly, it does not say it is invalid. 3.4.3-2 says:

"Possible undefined behavior ranges from ignoring the situation completely with unpredictable results, to behaving during translation or program execution in a documented manner characteristic of the environment (with or without the issuance of a diagnostic message), to terminating a translation or execution (with the issuance of a diagnostic message)."

Reply:
```
cstuff1.c:260:7: error: label at end of compound statement
  260 |     { L2: }
      |       ^~
```
This looks like a genuine parser bug, but it's now valid as part of C2X 6.8.2.